src/database/mutations/ApprovedItem.ts

```diff
@@ -21,6 +21,9 @@ export async function createApprovedItem(
   return db.approvedItem.create({
     data: {
       ...data,
+      // Make sure the language code is in upper case.
+      language: data.language?.toUpperCase(),
```

Comment:
Wondering why we need to make it to upper case here if the helper function checks in the resolver that we only receive the correct language code i.e uppercase EN/DE